hivemind/p2p/p2p_daemon.py

```diff
@@ -165,7 +167,11 @@ async def create(
         ]:
             if value:
                 process_kwargs[param] = self._maddrs_to_str(value)
+
         if identity_path is not None:
+            if not os.path.isfile(identity_path):
```

Comment:
[nit] there may be a situation where not only file is missing, but the preceding directory is also nonexistent. Here's what can happen:
- create folders (os.mkdirs(..., exist_ok=True))
- raise an explicit error message ("e.g. parent directory does not exist, not "file not found"")

Reply:
I've added the explicit error message to `P2P.generate_identity_file()`.